book/website/communication/aa/overview.md

```diff
@@ -0,0 +1,54 @@
+(communication-aa-overview)=
+# Overview of Academic Authorship
+
+A great deal of value is placed on academic authorship because a peer-reviewed journal article is still seen as the most valued output from a research project. 
+The number of articles an academic researcher is named on, and particularly whether the person is named as first author, has significant impact on their career.  
+
+But who should be an author on an article? And can we move away from the idea of first author taking the most credit?
+
+(communication-aa-overview-who)=
+## Who is an author? or are they actually contributors?
+
+Assigning names to a publication can be a tricky business. 
+However, an important decision that needs to be made at the outset of a project is whether the names on the articles produced from the project are going to be authors or contributors, or a combination of both. 
+
+The decision may essentially be out of your hands as journals and publishers have specific policies on authorship criteria. 
+Therefore, if you want to publish in a specific journal you will have to adhere to the policy and make an agreement about this within your project.
```

Comment:
Maybe here or somewhere else we can add this information regarding some publishers try to make these authorship models more justified such as PAS require public disclosure regarding the contributions of each author as in [here](https://www.pnas.org/authors/editorial-and-journal-policies) or some asks for the creation of a database that track the authors's contributions as explained [here](https://www.nature.com/articles/489475a) via [altmetrics](http://altmetrics.org/manifesto/) and some require a public correspondent for the whole article that takes all the responsibility for such. Also the use of ORCID Identifier is again accepted by some publishers that would increase the traceability that might be lost due to the name, affiliation etc. changes.